chainerui/models/result.py

```diff
@@ -43,17 +44,60 @@ def __init__(self, path_name=None, name=None, project_id=None,
     def __repr__(self):
         return '<Result id: %r, path_name: %r />' % (self.id, self.path_name)
 
-    @property
-    def serialize(self):
-        """serialize."""
+    @classmethod
+    def create(cls, path_name=None, name=None, project_id=None,
+               log_modified_at=None):
+        """Initialize an instance and save it to db."""
+        result = cls(path_name, name, project_id, log_modified_at)
+
+        DB_SESSION.add(result)
+        DB_SESSION.commit()
+
+        from chainerui.tasks import crawl_result
```

Comment:
I put `import crawl_result` right before I use `crawl_result`.
This is a workaround for an issue in [this comment](https://github.com/chainer/chainerui/pull/92/files/2bdcc1534669316ea75857f50502f1aa45931c63#diff-f44eb8d0fdf1cd41d4318f69391a3dd0R57).

ChainerUI seems to use this workaround in other places too, but I am not sure this is a proper workaround. I will continue looking for the cause of this `import` issue after this PR is merged.

Reply:
When put `import crawl_result` on the top of `result.py`, modules are loaded on the bellow order.

1. call `chainerui/tasks/__init__.py` (not complete to load `crawl_result` as function yet on this timing)
2. call `chainerui/tasks/crawl_result.py`
3. call `chainerui/models/result.py`
4. call `chainerui/task/crawl_result.py` as `from chainerui.tasks import crawl_result`

So, `result.py` bounds `crawl_rresult` as module, before `from chainerui.tasks.craw_result import crawl_result` is not completed on `chainerui/tasks/__init__.py`.

Reply:
Thank you for the investigation.
It seems to be difficult to completely solve this issue, so I want to merge this PR for now and work on this issue in the future.